packages/cli/src/commands/account/claims.test.ts

```diff
@@ -0,0 +1,102 @@
+import { IdentityMetadataWrapper, newKitFromWeb3 } from '@celo/contractkit'
+import { ClaimTypes } from '@celo/contractkit/lib/identity'
+import { readFileSync, writeFileSync } from 'fs'
+import { tmpdir } from 'os'
+import Web3 from 'web3'
+import { testWithGanache } from '../../test-utils/ganache-test'
+import { commandExists } from '../../utils/exec'
+import ClaimDomain from './claim-domain'
+import ClaimKeybase from './claim-keybase'
+import ClaimName from './claim-name'
+import CreateMetadata from './create-metadata'
+import RegisterMetadata from './register-metadata'
+process.env.NO_SYNCCHECK = 'true'
+
+testWithGanache('account:authorize cmd', (web3: Web3) => {
+  let account: string
+
+  beforeEach(async () => {
+    const accounts = await web3.eth.getAccounts()
+    account = accounts[0]
+  })
+
+  describe('Modifying the metadata file', () => {
+    const emptyFilePath = `${tmpdir()}/metadata.json`
+    const generateEmptyMetadataFile = () => {
+      writeFileSync(emptyFilePath, IdentityMetadataWrapper.fromEmpty(account))
+    }
+
+    const readFile = () => {
+      return IdentityMetadataWrapper.fromFile(emptyFilePath)
+    }
+
+    test('account:create-metadata cmd', async () => {
+      const newFilePath = `${tmpdir()}/newfile.json`
```

Comment:
don't you need to remove the file if already present?

Reply:
Yeah, but I'm just assuming that it does not exist :)